programs/rodeos/ship_client.hpp

```diff
@@ -32,7 +33,7 @@ struct connection_callbacks {
    virtual bool received(ship::get_blocks_result_v0& result, eosio::input_stream bin) { return true; }
    virtual bool received(ship::get_blocks_result_v1& result, eosio::input_stream bin) { return true; }
    virtual bool received(ship::get_blocks_result_v2& result, eosio::input_stream bin) { return true; }
-   virtual void closed(bool retry) = 0;
+   virtual void closed(bool retry, bool quititingi = false) = 0;
```

Comment:
spelling of quitting.

Reply:
I would prefer for `quitting` not to be defaulted.

Reply:
Sorry for the typo. Made it not defaulted.